portfolio/src/main/webapp/planet.html

```diff
@@ -10,23 +10,23 @@
     <div id="content">
       <h1>Helping Our Planet</h1>
       <section>
-        <p>One thing I'm super passionate about is the environment and helping to mitigate
+        <p>One thing I'm incredibly passionate about is the environment and helping to mitigate
             climate change. </p>
-        <p>Below are just a few lifestyle changes I have made, and that 
-            anyone can make, to be more sustainable! (Hover over the images to learn more :))
+        <p>Below are just a few lifestyle changes that I have made, and that 
+           everyone can work on, to be more sustainable! (Hover over the images to learn more :))
         </p>
         <div class="planet-imgs">
-          <img src="images/vegan.png"/>
```

Comment:
If you're not using this image anymore you should remove it. Same goes for clothes.png and reusable.png.

Reply:
good point, I just fixed this! Thank you